pages/App/Home.js

```diff
@@ -0,0 +1,76 @@
+import Image from 'next/image'
+import { useState, useRef, useEffect } from 'react';
+import { FormInput } from '../../components/shared/formElements';
+import { required } from '../../utility/validation-rules';
+
+export default function Home() {
```

Comment:
Try to use `getStaticProps`, `getServerSideProps` methods

Reply:
resolved
Added getStaticProps and getServerSideProps methods

Reply:
resolved
Added getStaticProps and getServerSideProps methods

Reply:
Where is `getServerSideProps` method is used?

Reply:
resolved